pkgs/development/python-modules/ansible-kernel/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, writeText
+, ipywidgets
+, six
+, docopt
+, tqdm
+, jupyter
+, psutil
+, pyyaml
+, ansible-runner
+, ansible
+, python
+}:
+
+let kernelSpecFile = writeText "kernel.json" (builtins.toJSON {
+       argv = [ "${python.interpreter}" "-m" "ansible_kernel" "-f" "{connection_file}" ];
+       codemirror_mode = "yaml";
+       display_name = "Ansible";
+       language = "ansible";
+    });
+in
+buildPythonPackage rec {
+  pname = "ansible-kernel";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "a59039a1724c0f4f4435316e2ad3383f2328ae61f190e74414a66cc8c4637636";
+  };
+
+  propagatedBuildInputs = [ ipywidgets six docopt tqdm jupyter psutil pyyaml ansible-runner ansible ];
+
+  postPatch = ''
+   touch LICENSE.md
```

Comment:
FYI: https://github.com/ansible/ansible-jupyter-kernel/pull/82

Reply:
added note to update when your Pr is mereged. Fixed.